src/components/Book.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './Book.css';
+import PropTypes from 'prop-types';
+
+function Book({ singleBook }) {
+  return (
+    <div className="bookRow">
+      <div className="bookInfo">
+        <p className="category">{singleBook.category}</p>
+        <h1 className="title">
+          {' '}
+          {singleBook.title}
+        </h1>
+        <p className="author">{singleBook.author}</p>
+      </div>
+      <div className="actions">
+        <ul>
+          <li>Comments</li>
+          <li>Remove</li>
+          <li>Edit</li>
+        </ul>
+      </div>
+    </div>
+  );
+}
+
+Book.propTypes = {
+  singleBook: PropTypes.string.isRequired,
+};
+
```

Comment:
- [OPTIONAL] Your `singleBook` is an object. For these variables that are a set of something. The prop types should be more descriptive. As you know your prop has some keys like {title, author, category}. So you want to consider using this:  😉 

```js
Book.propTypes = {
  singleBook: PropTypes.shape({
    title: PropTypes.string,
    author: PropTypes.string,
    category: PropTypes.string,
  }).isRequired,
};
```

(If you don't do that the browser's console will show console errors.)

Reply:
Thank you 😁